dashboard/app/views/shared/_user_header.html.haml

```diff
@@ -1,5 +1,4 @@
 - if current_user
-  .transparent_backdrop{style: 'display:none;'}
```

Comment:
Removed in https://github.com/code-dot-org/code-dot-org/pull/1087 but I missed this one.